WordPress/Sniffs/WP/DeprecatedParameterValuesSniff.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPress\Sniffs\WP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Check for usage of deprecated parameter values in WP functions and provide alternative based on the parameter passed.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   1.0.0
+ *
+ * @uses    \WordPress\Sniff::$minimum_supported_version
+ */
+class DeprecatedParameterValuesSniff extends AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'wp_deprecated_parameter_values';
+
+	/**
+	 * Array of function, argument, and replacement function for deprecated argument.
+	 *
+	 * The list of deprecated parameter values can be found by
+	 * looking for `_deprecated_argument()`, `apply_filters_deprecated()`
+	 * and `do_action_deprecated()`.
+	 * The list is sorted first by the WordPress version when deprecated
+	 * and then alphabetically.
+	 * Last updated for WordPress 4.9.6.
```

Comment:
Just wondering: how complete is this list at this moment ?

Reply:
I re-checked it yesterday.